File: linear-search.py
list = [10, 20, 30, 40, 50]

def linear_search(target):
  for item in list:
    if item == target:
      print(f"{item} is the target.")
      return
    else:
      print(f"{item} is not the target.")

linear_search(30)

File: linear-search.js
let list = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]

function linear_search(target) {
  for (let i = 0; i < list.length; i++) {
    if (target === list[i]) {
      return `${target} está na posição ${i}`
    }
  }
}

console.log(linear_search(4))